projects.go

```diff
@@ -48,6 +48,7 @@ type Project struct {
 	HTTPURLToRepo                             string                     `json:"http_url_to_repo"`
 	WebURL                                    string                     `json:"web_url"`
 	ReadmeURL                                 string                     `json:"readme_url"`
+	Topics                                    []string                   `json:"topics"`
```

Comment:
These fields are ordered using to ordering in the docs. For this field that means it should be placed between just below the `TagList` field.

Reply:
Done, thanks!